website/src/main/tut/docs/users/configuration.md

```diff
@@ -17,20 +17,30 @@ The convention is to keep a file `.scalafix.conf` into the root directory of you
 Configuration is not needed or is optional for most rules, so you may not need to create a `.scalafix.conf`.
 
 ### rules
-Configure which rule to run with `rules = [ ... ]`.
-Scalafix comes with a small set of built-in rules.
+Configure which rule to run with `rules = [ ... ]`, for example
 
 ```scala
 // Built in rules
 rules = [
-{% for rule in site.data.rules.rules %}  {{ rule }}
-{% endfor %}]
+  // pre-installed rule from scalafix
+  ExplicitResultTypes
+  // custom rule on local disk
+  "file:rules/MyRule.scala"
+  // custom rule shared as a gist
+  "https://gist.githubusercontent.com/olafurpg/24607c9832d2f2c4b8b18aef89125061/raw/e60375343cc0039cca85da2457755fc466b4d616/libz.scala"
```

Comment:
???

Reply:
I guess this is an example of a rule shared as a gist (see the comment above)

Reply:
Hum, this is a bit obscure. Why not a nice url like: `https://example.com/rewrite`. I think there is git.io that makes github urls look nice.

Reply:
https://git.io/vNXDG

Reply:
https://github.com/blog/985-git-io-github-url-shortener

Reply:
`git.io` urls are shorter but they are still not memorable and you have to copy-paste them, a bare-bones URL to a raw gist give more information about what the url represents

Reply:
I prefer to keep the gist url since you don't have to visit the url to guess what it points to.

Reply:
I think it does not give this feature it's just value. It's useful for not having to publish your rule to maven central, but having a long complex url like this make it look hard.

Reply:
Fair enough, I updated to the shortened url.

Reply:
I agree with you that git.io hides the `libz.scala` but compared to the other rules:
```
rule = "github:typelevel/cats/v1.0.0"
```

```
rule = https://gist.githubusercontent.com/olafurpg/24607c9832d2f2c4b8b18aef89125061/raw/e60375343cc0039cca85da2457755fc466b4d616/libz.scala
```

```
rule = https://git.io/vNXDG
```

Reply:
@olafurpg :heart: thanks for your patience.